composer.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "friendsofphp/number",
+    "description": "Adds a Number utility class",
```

Comment:
It looks like a commit message 😉. Maybe something more like:

```suggestion
    "description": "Number utility for displaying human-friendly numeric data",
```

Reply:
Perfect, thanks!